e2e/transform/transform-testrunner/test-runner.ts

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+import {Config} from '@jest/types';
+import {JestEnvironment} from '@jest/environment';
+import Runtime from 'jest-runtime';
+import {TestResult} from '@jest/test-result';
+
+export default function testRunner(
+  globalConfig: Config.GlobalConfig,
+  config: Config.ProjectConfig,
+  environment: JestEnvironment,
+  runtime: Runtime,
+  testPath: string
+): Promise<TestResult> {
+  return Promise.resolve({
```

Comment:
just make the function `async` rather than `Promise.resolve`?

Reply:
we really should have a helper for creating "empty" test results. This is a pain for custom runners (as you can see here)